src/components/MainPageMovieContainer.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react'
+import { Card, Container, Image, Grid } from 'semantic-ui-react'
+import axios from 'axios'
+import _ from 'lodash'
+import he from 'he'
```

Comment:
what is this?

Reply:
he is used in this application to decode the HTML entities we have received in our api call. 

https://github.com/mathiasbynens/he